packages/react/src/themes/teams/components/Dropdown/dropdownVariables.ts

```diff
@@ -1,46 +1,74 @@
 import { pxToRem } from '../../../../lib'
 export interface DropdownVariables {
   backgroundColor: string
+  backgroundColorHover: string
+  borderColor: string
   borderColorFocus: string
   borderRadius: string
+  openBorderRadius: string
   borderWidth: string
+  searchBorderBottomWidth: string
   color: string
   comboboxPaddingButton: string
   comboboxFlexBasis: string
   listBackgroundColor: string
+  listBorderColor: string
   listBorderRadius: string
+  listBorderWidth: string
   listPadding: string
   listBoxShadow: string
   listMaxHeight: string
   listItemBackgroundColor: string
+  listItemColorHover: string
+  listItemBackgroundColorHover: string
   listItemBackgroundColorActive: string
   listItemColorActive: string
+  listItemSelectedFontWeight: number
+  listItemSelectedColor: string
+  selectedItemColor: string
+  selectedItemBackgroundColor: string
+  selectedItemColorFocus: string
   selectedItemBackgroundColorFocus: string
   selectedItemsMaxHeight: string
   toggleIndicatorSize: string
+  triggerButtonColorHover: string
   width: string
 }
 
 const [_2px_asRem, _3px_asRem, _12px_asRem] = [2, 3, 12].map(v => pxToRem(v))
 
 export default (siteVars): DropdownVariables => ({
   backgroundColor: siteVars.colors.grey[100],
+  backgroundColorHover: siteVars.colors.grey[150],
+  borderColor: 'transparent',
   borderColorFocus: siteVars.colors.brand[600],
   borderRadius: `${_3px_asRem} ${_3px_asRem} ${_2px_asRem} ${_2px_asRem}`,
-  borderWidth: `0 0 ${pxToRem(2)} 0`,
+  openBorderRadius: `${_3px_asRem} ${_3px_asRem} 0 0`,
+  borderWidth: '0px',
+  searchBorderBottomWidth: pxToRem(2),
   color: siteVars.bodyColor,
+  selectedItemColor: siteVars.bodyColor,
   comboboxPaddingButton: `0 ${_12px_asRem}`,
   comboboxFlexBasis: pxToRem(50),
   listBackgroundColor: siteVars.colors.white,
-  listBorderRadius: _3px_asRem,
+  listBorderRadius: `0 0 ${_3px_asRem} ${_3px_asRem}`,
+  listBorderColor: 'transparent',
+  listBorderWidth: '0px',
   listPadding: `${pxToRem(8)} 0`,
   listBoxShadow: `0 .2rem .6rem 0 ${siteVars.colors.grey[250]}`,
```

Comment:
The `listBoxShadow` looks wrong
![image](https://user-images.githubusercontent.com/828692/57589140-66513e80-74d4-11e9-8db6-28720dcae3ff.png)

Reply:
This was there before, so I wasn't sure whether it should be changed. Will check the designs again.

Reply:
This is fixed now, please take another look.